src/font/text.hpp

```diff
@@ -157,6 +157,14 @@ class pango_text
 	point get_cursor_position(
 		const unsigned column, const unsigned line = 0) const;
 
+	/**
+	 * Get maximum length.
+	 * 
+	 * @returns                   The maximum length of the text. The length of text
+	 *                            should not over this value.
```

Comment:
Grammatically incorrect. This would be correct:

"The maximum length of the text. The length of text should not **exceed** this value."